client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -26,9 +25,8 @@ const ChecklistItem = ( { task }: { task: Task } ) => {
 			<Button
 				className="launchpad__checklist-item"
 				disabled={ taskDisabled }
-				href={ actionUrl }
 				data-task={ id }
-				{ ...action }
+				onClick={ actionDispatch }
```

Comment:
I've completely removed the `actionUrl` (`<a>` tag with `href`) in favor of `actionDispatch` (`<button>` with `onClick`). We want to convert all tasks to have a click handler so we can record the tracks event on click.